google/resource_container_cluster_test.go

```diff
@@ -921,6 +961,30 @@ resource "google_container_cluster" "with_master_auth" {
 	}
 }`, acctest.RandString(10))
 
+var testAccContainerCluster_withNetworkPolicyEnabled = fmt.Sprintf(`
+	resource "google_container_cluster" "with_network_policy_enabled" {
+		name = "cluster-test-%s"
+		zone = "us-central1-a"
+		initial_node_count = 1
+	
+		network_policy {
+			enabled = true
+			provider = "CALICO"
+		}	
+	}`, acctest.RandString(10))
+
+var testAccContainerCluster_updateNetworkPolicyEnabled = fmt.Sprintf(`
+		resource "google_container_cluster" "with_network_policy_enabled" {
+			name = "cluster-test-%s"
```

Comment:
Since this config and the previous one both use random strings as part of the cluster name, this will destroy+recreate the cluster so it's not actually testing update.